melos.yaml

```diff
@@ -4,6 +4,10 @@ packages:
   - packages/**
 
```

Comment:
:bulb:

Reply:
@Salakar I cannot get a global ignore to work in this way at all

I add:

```
ignore:
  - "*firebase_ml_custom*"
```

...at the top of the file and it has zero affect on `melos run build:example_ios` whereas it works on the target. Just for grins I tried ignores of `*`, `**`, `*firebase*` etc just to see if the global ignore would have any influence at all and it did not seem to ever change the total package count. It would change `melos list` output though. :thinking: :shrug: 

Is there something interacting with application of global ignore such that it wouldn't work?

After I repush this (in a few minutes after grooming what I can) you might pull this branch and try it.

Reply:
after carefully checking selection criteria, it seems I only need to have this in a couple targets, since global ignore wasn't working, I put it in where needed and it works locally again.